<launch>
  <node pkg="eye_process" name="left_eye" type="vision_node_center.py" output="screen">
    <param name="camera_name" value="left_cam" />
    <param name="left_right" value="0" />
    <param name="red_threshold" value="250" />
    <param name="receptive_file_dir" value="$(find eye_process)/json/receptive_field.json" />
    <param name="pub_freq" value="20" />
  </node>

   <node pkg="eye_process" name="right_eye" type="vision_node_center.py" output="screen">
    <param name="camera_name" value="right_cam" />
    <param name="left_right" value="1" />
    <param name="red_threshold" value="250" />
    <param name="receptive_file_dir" value="$(find eye_process)/json/receptive_field.json" />
    <param name="pub_freq" value="20" />
  </node>
</launch>
